Firestore/core/test/firebase/firestore/remote/exponential_backoff_test.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <chrono>  // NOLINT(build/c++11)
+
+#include "Firestore/core/src/firebase/firestore/remote/exponential_backoff.h"
+#include "Firestore/core/src/firebase/firestore/util/async_queue.h"
+#include "Firestore/core/src/firebase/firestore/util/executor_std.h"
+#include "Firestore/core/test/firebase/firestore/util/async_tests_util.h"
+#include "absl/memory/memory.h"
+#include "gtest/gtest.h"
+
+using firebase::firestore::util::AsyncQueue;
```

Comment:
I recently read http://go/totw/119 which suggests moving these using declarations inside the namespace statements. We're not at all consistent with this though. (If anything, we more consistently don't do this.)

Optional.

Reply:
Done.

I think adding a leading `::` is a bit overkill, though. What do you think?

Reply:
For sibling namespaces, that link suggests that fully qualifying (with a leading `::`) is always correct, but does allow for relative usage as well. Looking at this specific case, it seems unlikely that we'd introduce any of these namespaces:

```
::firebase::firebase
::firebase::firestore::firebase
::firebase::firestore::remote::firebase
```
(and if we ever do, we've got bigger problems).

So I think dropping the `::` would be fine. OTOH, including it would allow us to never have to stop and think about it.

FWIW, after I read that link, I starting moving my `using` declarations inside of my namespaces, but I didn't add leading `::`s. Probably worth kicking this over to the chat and quickly deciding as a group what we want to do. (Though we've already spent more time discussing it in this PR than it's worth. This is a pretty minor issue.)

For the purposes of this PR, I'd recommend doing whatever you like. If we decide to "fix" it, we can do so all at once later.